JSDemos/Demos/Charts/Area/description.md

```diff
@@ -1 +1,12 @@
-Area series can be&nbsp;used to&nbsp;clearly emphasize change in&nbsp;values. In&nbsp;this example, you can switch between the _area_, _stacked area_ and _full-stacked area_ series types. _Stacked area_ series help visualize multiple series and compare the manner in&nbsp;which each series contributes to&nbsp;the total aggregate value for specific arguments. _Full-stacked area_ series help compare the percentage value of&nbsp;multiple area series for each argument.
\ No newline at end of file
+The Chart component can display various [series types](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type). This demo shows the following series types (use a drop-down editor below the Chart to select the type):
+
+- *area*    
+Area series visualize data as an area filled with color. If the Chart contains multiple area series, they overlap each other.
+
+- *stackedarea*    
+In stacked area series, the value of each next point is counted off from the previous point with the same argument. As a result, The Chart displays areas stacked on top of each other without overlapping. This type of Chart is useful when you need to show a ratio between different areas.
```

Comment:
The word "stack" should be clear enough in the description. No need to describe things like "each next point is counted off from the previous point". 

```suggestion
The Chart displays areas stacked on top of each other without overlapping. This type of Chart is useful if you need to compare contributions of each series to the overall value.
```